### ISSUE TITLE
Configure Zowe as a started task

### ISSUE THREAD

**Issue by John-A-Davies** [zowe/zowe-install-packaging#101]
The doc tells you what to do manually, but it would be better if the install script did it instead:

From the User Guide PDF 

```The ZOWESVR must be configured as a started task (STC) under the IZUSVR user ID.
• If you use RACF, issue the following commands:
RDEFINE STARTED ZOWESVR.* UACC(NONE) STDATA(USER(IZUSVR) GROUP(IZUADMIN)
PRIVILEGED(NO) TRUSTED(NO) TRACE(YES))
SETROPTS REFRESH RACLIST(STARTED)
• If you use CA ACF2, issue the following commands:
SET CONTROL(GSO)
INSERT STC.ZOWESVR LOGONID(IZUSVR) GROUP(IZUADMIN) STCID(ZOWESVR)
F ACF2,REFRESH(STC)
• If you use CA Top Secret, issue the following commands:
TSS ADDTO(STC) PROCNAME(ZOWESVR) ACID(IZUSVR)
```

**Comment by Joe-Winchester** [comment]
@John-A-Davies - are you still intending to do this ?  I feel that this is a one off per site command so rather than it being the install script, it is something that could be done in a zowe-configure-system.sh script, so that the installer is just concerned about doing an install and system wide configurations are kept separate.  
Does the zowe-verify.sh script do any checking ?

**Comment by John-A-Davies** [comment]
@Joe-Winchester Yes, I have a script to do this.  Will test it and create PR.

**Comment by John-A-Davies** [comment]
Closed by #218
